.github/workflows/clear-all-data-dev.yaml

```diff
@@ -10,9 +10,9 @@ jobs:
     timeout-minutes: 10
     environment:
       name: dev
-
+  
     steps:
-      - name: Checkout
+      - name: checkout
```

Comment:
Nitpick but I previously attempted to make all this capitalization consistent :( Can we pick upper- or lowercase and use it for all of them?